flocker/node/agents/blockdevice.py

```diff
@@ -1503,12 +1518,17 @@ def calculate_changes(self, configuration, cluster_state, local_state):
         ))
         mounts = list(self._calculate_mounts(
             local_node_state.devices, local_node_state.paths,
-            configured_manifestations, local_state.volumes
+            configured_manifestations, local_state.volumes,
+            local_state.discovered_datasets,
```

Comment:
Does "discovered_datasets" exist as a parameter? Or should this just be "datasets"?

In summary, personal lesson learned about letting non-verified fakes into the codebase.